### ISSUE TITLE
Random cellular network drops on Pro 5

### ISSUE THREAD

**Issue by the-Mitu** [ubports/ubports-touch#156]
The phone tends to drop the entire GSM networks from time to time. It's much less frequent the dropping the data connection, but still it does happen that sometimes I don't have the connection when everyone around does. In most cases the connecition comes back without my actions.

When network drops, I see either "Rejected" or "Unregistered" text instead of carrier name in the network indicator's menu (texts may not be exact, i see them in Polish).

If you need any logs or additional information, please tell me what should I do to gather them.

**Comment by NeoTheThird** [comment]
Where's the difference to #155 ?

**Comment by the-Mitu** [comment]
#155 is about mobile *data* connection drops - but you still can make calls for example. This one is about the whole cellular network dropping - you have no GSM signal at all and you can neither browse Internet nor call. I think these are different situations.

**Comment by NeoTheThird** [comment]
I see. Sorry about that.

**Comment by mihaelmilea** [comment]
I don't have this issue on my Pro 5.

**Comment by the-Mitu** [comment]
I've found a few topics on XDA with different basebands for Pro5: https://forum.xda-developers.com/meizu-pro-5/development

I *think* that replacing the baseband with the other one might help, but it should be tested. I tried to swap the baseband myself, however it is not possible to just flash it via recovery on Ubuntu (i guess bacause it sits in the android LXC container).

could anyone test these basebands or guide me how to replace a baseband in my phone?

**Comment by Flohack74** [comment]
No, baseband flashing should be done from fastboot, and its a .img file. You cannot flash it from either recovery or the OS itself. Which file you have got?

**Comment by the-Mitu** [comment]
@Flohack74, Take a look here:
https://forum.xda-developers.com/meizu-pro-5/development/meizu-pro5-baseband-25-10-2017-t3699239

I've downloaded the .zip and tried to flash it via recovery as it is seen. I also tried to extract the zip to find out if I can swap the files manually - there is mainly a modem.bin inside and no .img files at all.

On Flyme they definitely have to flash basebands via recovery, so Ubuntu seems to be different here.

**Comment by Flohack74** [comment]
Ok I can see, they are using an embedded technique, that means we would have stuff it into our system.img - I will talk with the devs and come back to this. Problem is that we are currently not building this system.img for Meizu and BQ, so this can be quite tricky. And yes, you are right, it belongs inside the container. You can however mount the container file probably, and then replace it. But this needs some advanced skills ^^

**Comment by the-Mitu** [comment]
@Flohack74 
Could you tell me where to start? Maybe I can find some time sometime and try to look into it.

**Comment by Flohack74** [comment]
Ask the Halium guys, they are currently more skilled in all this porting stuff. I did not touch it for months, and cannot tell you right now what files to loop-mount, but for sure its system.img or ubuntu.img and then inside there is a folder with the lxc container to be mounted, too ;)

**Comment by the-Mitu** [comment]
I'll add this information here as well, as it's probably important: I wasn't aware that there are two different versions of Pro 5, but mine is TD-LTE.

**Comment by lduboeuf** [comment]
@the-Mitu is this issue still relevant ?

**Comment by wesertrainer** [comment]
The problem still remains for the TD-LTD Version of the Pro 5.
